src/js/events/MesosSummaryActions.js

```diff
@@ -104,12 +104,17 @@ var MesosSummaryActions = {
     function (resolve, reject) {
 
       return function (timeScale) {
-        let canAccessHistoryServer = Hooks.applyFilter(
+        let canAccessHistoryAPI = Hooks.applyFilter(
           'hasCapability', false, 'historyServiceAPI'
         );
 
-        if (!_historyServiceOnline || !canAccessHistoryServer) {
-          requestFromMesos(resolve, reject);
+        if (!_historyServiceOnline || !canAccessHistoryAPI) {
+          let canAccessMesosAPI = Hooks.applyFilter(
+            'hasCapability', false, 'mesosAPI'
+          );
+          if (canAccessMesosAPI) {
```

Comment:
I think we can fix this once we move to relay.
It's a shitty fix, but good enough IMO. Doesn't really add tech-debt, it just doesn't improve the current way of doing things.
We need this so that the UI can load when Alice only has access to Jobs or Universe but not to Mesos.